doc/source/examples/analysis/alignment_and_rms/aligning_trajectory.ipynb

```diff
@@ -21,6 +21,9 @@
     "    \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # import nglview as nv
</pre><p>Any specific reason for disabling the imports completely?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Just that if people don't want to install nglview, they don't need to.